tests/SprykerEcoTest/Zed/Heidelpay/Business/HeidelpayFacadeProcessNotificationTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Heidelpay\Business;
+
+use Generated\Shared\Transfer\HeidelpayNotificationTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Orm\Zed\Heidelpay\Persistence\SpyPaymentHeidelpayNotification;
+use Orm\Zed\Heidelpay\Persistence\SpyPaymentHeidelpayNotificationQuery;
+use Orm\Zed\Sales\Persistence\SpySalesOrder;
+use Propel\Runtime\Propel;
+use SprykerEco\Zed\Heidelpay\Business\HeidelpayBusinessFactory;
+use SprykerEco\Zed\Heidelpay\Business\HeidelpayFacade;
+use SprykerEcoTest\Zed\Heidelpay\Business\DataProviders\PaymentBuilder;
+use SprykerEcoTest\Zed\Heidelpay\Business\Mock\SuccessfulResponseHeidelpayBusinessFactory;
+use SprykerEcoTest\Zed\Heidelpay\Business\Mock\UnsuccesfulResponseHeidelpayBusinessFactory;
+
+/**
+ * @group Functional
+ * @group Spryker
+ * @group Zed
+ * @group Heidelpay
+ * @group Business
+ * @group HeidelpayFacadeAuthorizePaymentTest
```

Comment:
Please check it. Normally it should be fixed by codestyle fixer, but I'm not sure if it works for Eco modules